src/ameba/cli/cmd.cr

```diff
@@ -14,7 +14,7 @@ module Ameba::Cli
       raise "Invalid usage: Cannot explain an issue and autocorrect at the same time."
     end
 
-    config = Config.load opts.config, opts.colors?
+    config = Config.load path: opts.config, colors: opts.colors?, no_config: opts.no_config?
```

Comment:
passing no_config to Config module looks a bit weird. About about renaming no_config to something line "skip_reading_config" or "skip_global_paths"?

Reply:
Please avoid adding new CLI switches. This should be working without those.

Reply:
> Please avoid adding new CLI switches. This should be working without those.

Okay, i just thought add a new switch will make code more clearly.